src/appsignal/__init__.py

```diff
@@ -1,7 +1,32 @@
 from .client import Client as Appsignal
+from .helpers import (
+    set_body,
+    set_category,
+    set_custom_data,
+    set_header,
+    set_name,
+    set_namespace,
+    set_params,
+    set_root_name,
+    set_session_data,
+    set_tag,
+)
 
 
-__all__ = ["Appsignal"]
+__all__ = [
+    "Appsignal",
+    "set_body",
+    "set_category",
+    "set_custom_data",
+    "set_header",
+    "set_name",
+    "set_namespace",
+    "set_params",
+    "set_root_name",
+    "set_session_data",
+    "set_tag",
+]
```

Comment:
Oh we can export things on the "main" module.

Should we do the same for the metrics helpers in #127 or should this use that PR's approach of having to import using `appsignal.tracing`?

Reply:
I'm happy either way! I'm exposing them on the main module because this is what we do on Ruby and Node.js, where it's `Appsignal.set_error` and `import { setError } from "@appsignal/nodejs"`. But we don't need to do that.

Whatever we choose to do, let's do the same for these helpers and for the metrics helpers.

Reply:
Let's export it on the `__all__` thing so it's all on the same import. I assume that if people find the collection of helpers annoying to import, they can also import `appsignal.tracing` and call `tracing.set_...` like it's done now for the metrics helpers.

I'll also update the metric helpers to export them this way, but after your PR is merged.